### ISSUE TITLE
Feature request: Highlight line of code causing runtime panic

### ISSUE THREAD

**Issue by XsongyangX** [rust-lang/rust-analyzer#13636]
When the code crashes, I see a large backtrace with the panic statement somewhere in the console. It would be nice if the code editor also marks the lines where the panic happens.

This would help unit testing as well.

**Comment by bjorn3** [comment]
Can you post the backtrace? Panics of rust-analyzer are always bugs and in many cases it is not possible to pin point a single piece of user code that causes the panic.

**Comment by lnicola** [comment]
Sounds like this is about panics in the user programs, when started by RA. I think you can click on the path in the output window, but I might be wrong.

**Comment by XsongyangX** [comment]
> Sounds like this is about panics in the user programs, when started by RA. I think you can click on the path in the output window, but I might be wrong.

Yes, you can click on the path in the output window. It would be better if the editor highlights the lines in the file. This is especially helpful when dealing with failing unit & integration tests.

**Comment by ChayimFriedman2** [comment]
Probably irrelevant by now.
